gradle.properties

```diff
@@ -48,7 +48,7 @@ mod_name=Viner
 # The license of the mod. Review your options at https://choosealicense.com/. All Rights Reserved is the default.
 mod_license=MIT License
 # The mod version. See https://semver.org/
-mod_version=1.4.0
+mod_version=1.5.0
```

Comment:
With the addition of the Multiplayer fixes (thanks) I'm planning to make the last PR. this one, as well as some more tweaks on my end and releasing it on Modrinth as 2.0.0. 

If you have a modrinth profile let me know so I can credit you in the release notes!